audio-worklet/src/noisegate-script-processor.js

```diff
@@ -98,7 +106,7 @@ class NoiseGate {
   onaudioprocess_(event) {
     let inputBuffer = event.inputBuffer;
     let channel0 = inputBuffer.getChannelData(0);
-
+    
```

Comment:
Sorry.  Remove the extraneous spaces on this line, leaving a single blank line without spaces in it.